client/src/planwise/client/sources/components/dropdown.cljs

```diff
@@ -10,9 +10,13 @@
 
 (rf/reg-sub
```

Comment:
@ggiraldez I'd like to have a second opinion about the sources dropdown. I moved the filter by type from`sources-dropdown-component` to the subscription `:sources/dropdown-options`, while doing that I used `:sources/list` instead of directly reading from the `db`, removed calls to `asdf/should-reload?`.